README.md

```diff
@@ -99,22 +99,21 @@ If the type-check succeeded the application is executed, otherwise you will get
 error with the types that mismatch.
 
 In order to use this functionality you can use the `loadSafeFile` which takes the same
-configuration value as the `loadFile` function. Also, you need to have a `.scheme.yml`
+configuration value as the `loadFile` function. Also, you need to have a `.schema.yml`
 in your current directory. This file must have the following structure:
 
 ```yaml
-- type: bool
-  envs:
-    - name: DOTENV
-      required: true
-    - name: OTHERENV
-      required: false
-- type: integer
-  envs:
-    - name: PORT
-      required: true
-    - name: TOKEN
-      required: true
+- name: DOTENV
```

Comment:
@CristhianMotoche is there any check to prevent `EnvVaribles` duplication?

```yaml
- name: PORT
  type: bool
- name: PORT
  type: numeric
```

Reply:
@sestrella Currently, we don't have. However, I was checking that the system has the current behaviour: it matches the types with the value and then type checks for all the types specified.

For instance, in your example the value of `PORT` will be evaluated as a `bool` and `numeric`. The result from that will always fail. I think, this could be enough. WDYT?

Reply:
@CristhianMotoche do you mind adding an issue, so we could tackle that later, please?